app/js/buildProjects.js

```diff
@@ -1,6 +1,6 @@
-const projects = {
-  greenHouse: {
-    dataId: "greenHouse",
+let projects = [
```

Comment:
Why did you change this?

Reply:
Made the object into an array so appending it is less of a task